### PR TITLE
Clean up some code

### PR DESCRIPTION
**Pull request by AlexandreAbraham**
Very small modifications of code I ran into.

### CODE REVIEW

File: nilearn/plotting/img_plotting.py [hunk 1]
```diff
@@ -321,8 +315,8 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
 
 def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
               output_file=None, display_mode='ortho', figure=None,
-              axes=None, title=None, annotate=True, threshold=None, draw_cross=True,
-              black_bg='auto', dim=False, cmap=plt.cm.gray,
+              axes=None, title=None, annotate=True, threshold=None,
+              draw_cross=True, black_bg='auto', dim=False, cmap=plt.cm.gray,
```
Comment (AlexandreAbraham) [inline nilearn/plotting/img_plotting.py]: PEP8

File: nilearn/_utils/ndimage.py [hunk 2]
```diff
@@ -38,3 +38,12 @@ def largest_connected_component(volume):
...
 
+def get_border_data(data, border_size):
+    return np.concatenate([
+        data[:border_size, :, :].ravel(),
+        data[-border_size:, :, :].ravel(),
+        data[:, :border_size, :].ravel(),
+        data[:, -border_size:, :].ravel(),
+        data[:, :, :border_size].ravel(),
+        data[:, :, -border_size:].ravel(),
+    ])
```
Comment (AlexandreAbraham) [inline nilearn/_utils/ndimage.py]: For code factorization.

File: nilearn/plotting/img_plotting.py [hunk 3]
```diff
@@ -265,53 +266,46 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
     """
     vmin = None
     vmax = None
-    if anat_img is not False and anat_img is not None:
```
Comment (AlexandreAbraham) [inline nilearn/plotting/img_plotting.py]: No code modification, just reordering stuff in order to simplify the code.

**Comment by GaelVaroquaux** [comment]
LGTM. Merging (just in time for 0.1.3)

**Comment by AlexandreAbraham** [comment]
Well, I waited for this one because I did not want to merge last minute stuff in 0.1.3... But anyway, there is modification of logic so there should be no problem ;)

**Comment by GaelVaroquaux** [comment]
> Well, I waited for this one because I did not want to merge last minute stuff
> in 0.1.3...

I am still not finished releasing. Hopefully next hour.

> But anyway, there is modification of logic so there should be no
> problem ;)

I looked at the code twice. Another review would good.

File: nilearn/plotting/img_plotting.py [hunk 4]
```diff
@@ -265,53 +266,46 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
...
+        ptp = .5 * (vmax - vmin)
+        if black_bg:
+            if not isinstance(dim, numbers.Number):
+                dim = .8
+            vmax = vmean + (1 + dim) * ptp
+        else:
+            if not isinstance(dim, numbers.Number):
+                dim = .6
+            vmin = vmean - (1 + dim) * ptp
     return anat_img, black_bg, vmin, vmax
```
Comment (bthirion) [inline nilearn/plotting/img_plotting.py]: This part is slightly tricky, and will probably not fit every use case. On the other hand, explaining the logic of this code is harder than reading it.

**Comment by bthirion** [comment]
I don't see any issue by eye...

**Comment by GaelVaroquaux** [comment]
> I don't see any issue by eye...

Thanks for the review, @bthirion!

**Comment by eickenberg** [comment]
I can't see anything wrong with this either

**Pull request merged**